uber/reports.py

```diff
@@ -0,0 +1,48 @@
+from uber.common import *
```

Comment:
I'd be inclined to put all of this in ``utils.py`` since that's the generic place for random utility methods and classes such as this.  Not a huge deal, but I might move this stuff there when I get a chance.